Resources/Prototypes/Entities/Mobs/NPCs/animals.yml

```diff
@@ -2353,7 +2353,7 @@
       - ReagentId: Mechanotoxin
         Quantity: 1
   - type: MeleeChemicalInjector
-    transferAmount: 0.75
+    transferAmount: 3
```

Comment:
Please add a `Harmony change` comment to the change in the upstream yaml.

Reply:
Comment added.